recipes-core/images/xen-image-minimal.bb

```diff
@@ -0,0 +1,110 @@
+DESCRIPTION = "A minimal xen image"
```

Comment:
Is it a copy from meta-virtualization? It is not necessary, we should not copy it here.  INluding meta-virtualization in kas file should be enough.

Reply:
Resolved in 2c78eb80f059d7cffb40508604f5414b818ae989